react-app/src/app/screens/Game/components/Square/index.js

```diff
@@ -2,10 +2,12 @@ import React, { Component } from 'react';
 
 import style from './styles.scss';
 
-class Square extends Component {
-  render() {
-    return <button className={style.square}>{/* TODO */}</button>;
-  }
+function Square(props) {
```

Comment:
Destruct the object to use the attributes directly 
`function Square({ onClick, value })`